### ISSUE TITLE
Handle ANS Reviewer comments

### ISSUE THREAD

**Issue by katyhuff** [arfc/pride#91]
This issue can be closed with a PR that updates the reservoir computing paper in a manner that handles the review comments below. 

### Reviewer 1
- [ ] In Fig. 3, it appears that there are two local minima (rho = 0.9, noise = 10^-4; rho = 1.3, noise = 10^-2) which are both close to the global minimum. How did you select between these two? Since the two input parameters vary quite a bit between these two, I suspect that the output of the model using the other minimum would be significantly different, too.
- [ ] What more quantitative metrics could be used to evaluate the quality of forecasts? How many Lyapunov times are useful before some accuracy threshold is reached?

### Reviewer 2
- [ ] Interesting work. Might help to clearly state the objectives of this initial phase of research towards the beginning of the paper.

### Reviewer 3
- [ ] The summary is well written but needs some discussion on verification of the proposed methodology.
- [ ] What is the basis for selection of hyperparameters and some sensitivity and parametric analysis needs to be done in order to ensure confidence in results.
- [ ] How dependent is the methodology and results on the training length and network size ?

### Submit
- [ ] A suitably revised summary must be uploaded no later than August 17 into the same system that you submitted your original paper (https://epsr.ans.org/meeting/?m=308). If your revised summary is not received by the deadline date, your original summary will be published as is.
